front/src/applications/editor/tools/switchEdition/components.tsx

```diff
@@ -94,6 +94,15 @@ export const SwitchEditionLeftPanel = () => {
         }}
         onSubmit={async (flatSwitch) => {
           const entityToSave = flatSwitchToSwitch(switchType, flatSwitch as FlatSwitchEntity);
+          const detectedDuplicates = filter(
```

Comment:
if you filter first and then groupBy it should be less work

Reply:
I don't think it will work. I group by track, and then I'm looking for tracks that are used twice.